Makefile

```diff
@@ -1,2 +1,7 @@
 all:
 	bin/build.sh
+
+release: all
+	FILE_NAME="betaflight-tx-lua-scripts_$$(git describe --abbrev=0 --tags).zip"; \
+	rm -f $${FILE_NAME}; \
+	zip -r $${FILE_NAME} obj/
```

Comment:
Shouldn't we have something like:
```
cd obj/ && zip -r ../$${FILENAME} *
```

Otherwise, the files are located in `obj/*` in the archive. It seems easier to be able to unpack directly at the root of the SD card.

Reply:
Yeah we could - I just put it the way it is because that's how the zip files for all of the previous releases were organised. But we could change this, and change the instructions.

Re unpacking at the root of the SD card, I find that when doing this with the SD card inside the TX, mounted over USB, this results in the unpacking operation taking a very long time, probably due to the operating system (linux) copying the compressed files to the SD card, and then uncompressing them there. So I am not sure if I'd want to change the instructions to ask users to do it this way.

Reply:
Ok, got it. I never noticed it was done this way.